zScanner/ViewControllers/Login/LoginViewController.swift

```diff
@@ -142,13 +142,15 @@ class LoginViewController: BaseViewController, ErrorHandling {
             make.top.equalTo(titleLabel.snp.bottom).offset(20)
             make.centerX.equalToSuperview()
             make.right.left.equalToSuperview()
+            make.height.greaterThanOrEqualTo(30)
```

Comment:
Fix for LoginVC. The text field was jumping up and down a few pixels when you started typing. This was a legacy bug from the previous project.